### PR TITLE
Update

### PR DESCRIPTION
**Pull request by moshewe**

### CODE REVIEW

File: src/lokijs.js [hunk 1]
```diff
@@ -286,8 +286,8 @@
         }
 
         checkFn = containsCheckFn(a) || function () {
-          return false;
-        };
+            return false;
+          };
```
Comment (alexandernst) [inline src/lokijs.js]: wrong indent

File: src/lokijs.js [hunk 2]
```diff
@@ -307,8 +307,8 @@
 
         // return false on check if no check fn is found
         checkFn = containsCheckFn(a) || function () {
-          return false;
-        };
+            return false;
+          };
```
Comment (alexandernst) [inline src/lokijs.js]: wrong indent

File: src/lokijs.js [hunk 3]
```diff
@@ -772,12 +773,12 @@
...
-        return null;
-      default:
-        return value;
+        case 'autosaveHandle':
+        case 'persistenceAdapter':
+        case 'constraints':
+          return null;
+        default:
+          return value;
       }
```
Comment (alexandernst) [inline src/lokijs.js]: wrong indent

**Comment by alexandernst** [comment]
there are quite some indent issues. Please fix your editor and rebase your commits.

**Comment by moshewe** [comment]
my bad, was trying to update my fork

**Pull request closed without merging**